guides/common/modules/con_understanding-puppet-architecture.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Understanding_puppet_architecture_{context}"]
+= Understanding Puppet Architecture
+
+Puppet uses client-server architecture with managed hosts running the Puppet agent and {Project} or any {SmartProxies} acting as a Puppet server.
```

Comment:
more technically correct would be that Puppet server is the central component, where the definitions live. The {Project} or any {SmartProxies} are typically deployed with the Puppet server and {Project} acts as an External Node Classifier (ENC) for such Puppet server.